package.json

```diff
@@ -26,10 +26,7 @@
     "postversion": "git push origin master && git push origin master --tags",
     "pretest": "npm run build",
     "test": "mocha -r should",
-    "prebuild": "npm run build-node",
```

Comment:
> In addition, the server build will not require transpilation as the minimum supported NodeJS version (i.e. NodeJS 8) supports the use of ES6 language features as is.

Is this what you meant by the above?

Reply:
Yes, that is how the server build was performed prior to this commit.

Reply:
Ok, just making sure that had nothing to do with the NPM deploy.